internal/processor/filterhelper/filter_helper.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package filterhelper
+
+import (
+	"fmt"
+
+	"github.com/spf13/cast"
+
+	"github.com/open-telemetry/opentelemetry-collector/internal/data"
+)
+
+// NewAttributeValue is used to convert the raw `value` from ActionKeyValue to the supported trace attribute values.
+func NewAttributeValue(value interface{}) (data.AttributeValue, error) {
```

Comment:
Should this be part of the data package? Maybe renamed "NewAttributeValueRaw" or `FromRawValue`?

This was copied from `internal/processor/helper.go`